wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/Robot.java

```diff
@@ -62,7 +63,6 @@ public void disabledPeriodic() {
    */
   @Override
   public void autonomousInit() {
-    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
```

Comment:
I think it would be best to keep this here, in case robotContainer were modified to return a dynamic command (such as with a SendableChooser). In fact, since there are two Autonomous commands defined, this example might be a good place to use a SendableChooser.

Reply:
Done. Please re-review.